src/main/java/com/iridium/iridiumskyblock/gui/IslandBankGUI.java

```diff
@@ -54,6 +54,22 @@ public void onInventoryClick(InventoryClickEvent event) {
         Optional<BankItem> bankItem = IridiumSkyblock.getInstance().getBankItemList().stream().filter(item -> item.getItem().slot == event.getSlot()).findFirst();
         if (!bankItem.isPresent()) return;
 
+        if (event.isShiftClick()) {
```

Comment:
This is kinda weird, and you didnt delete the old code that was there origionally?